src/systems/userinput/resolve-action-sets.js

```diff
@@ -140,6 +145,7 @@ export function resolveActionSets() {
   userinput.toggleSet(sets.cursorHoldingPen, cursorHoldingPen);
   userinput.toggleSet(sets.cursorHoldingCamera, cursorHoldingCamera);
   userinput.toggleSet(sets.cursorHoldingInteractable, cursorHoldingInteractable);
+  userinput.toggleSet(sets.cursorHoldingUI, cursorHoldingUI);
```

Comment:
TODO: I was having issues getting this recognized with super-hands so didn't use this set yet. Not strictly necessary but want to figure out if the actions I'm polling are right and if I'm polling in the right spots